homeassistant/components/zwave_js/strings.json

```diff
@@ -15,13 +15,14 @@
       "install_addon": {
         "title": "The Z-Wave JS add-on installation has started"
       },
-      "start_addon": {
+      "configure_addon": {
```

Comment:
This will require us to rename the key in localize to not lose existing translations.

Reply:
Done.